packages/io-services-cms-models/src/utils/date-utils.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * Returns the ISOString rapresentation for a given UNIX timestamp.
+ *
+ * @param {Number} unixTimestamp - UNIX timestamp to be converted to ISOString
+ * @returns {String}
+ */
+const isoStringfromUnixTimestamp = (unixTimestamp: number): string =>
```

Comment:
**suggestion (non-blocking):** rename function or change description.

This function could be misunderstood and in some cases not work properly. The function says to translate the timestamp passed as input into ISO String, but in reality it currently translates it correctly only if the timestamp is expressed in milliseconds. If you pass it a timestamp expressed in seconds the translation will not be correct!
I suggest slightly modifying the name of fn to make it clearer, or better describe its behavior in the documentation.

Reply:
93edf2f6f8de40684f88f812d787b6a44a74a775